pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1594,6 +1602,37 @@ def _check_use_list_or_dict_literal(self, node: nodes.Call) -> None:
                 elif inferred.qname() == "builtins.dict" and not node.keywords:
                     self.add_message("use-dict-literal", node=node)
 
+    def _check_use_dict_literal(self, node: nodes.Call) -> None:
+        """Check if dict is created by using the literal {}."""
+        if not isinstance(node.func, astroid.Name):
+            return
+        inferred = utils.safe_infer(node.func)
+        if (
+            isinstance(inferred, nodes.ClassDef)
+            and inferred.qname() == "builtins.dict"
+            and not node.args
+            and node.keywords
+        ):
+            rebuilt_dict = []
+            for keyword in node.keywords:
+                if keyword not in node.kwargs:
+                    rebuilt_dict.append(f'"{keyword.arg}": {keyword.value.as_string()}')
+
+            for keyword in node.kwargs:
+                if isinstance(keyword.value, nodes.Name):
+                    rebuilt_dict.append(f"**{keyword.value.name}")
+
+            code_suggestion = f"{{{', '.join(rebuilt_dict)}}}"
+            short_suggestion = (
+                f"{code_suggestion[:100]}{'...' if len(code_suggestion) > 100 else ''}"
+            )
```

Comment:
Let's create a private function for rebuilding the dict, if we're going to truncate there's no need to create the whole ``rebuild_dict``. Also we could make the suggestion smaller (maybe 64 chars max) and cut in the middle of the string.

Something like this but in a private function:
```suggestion
            short_suggestion  = "{"
            for keyword in node.keywords:
                if keyword not in node.kwargs:
                    if len(short_suggestion) <= 64:
                        short_suggestion += f'"{keyword.arg}": {keyword.value.as_string()}'
            for keyword in node.kwargs:
                if isinstance(keyword.value, nodes.Name):
                    if len(short_suggestion) <= 64:
                        short_suggestion += f"**{keyword.value.name}"
            short_suggestion += "}"
            if len(short_suggestion) > 64                        
                short_suggestion = f"{short_suggestion[30]}...{short_suggestion[:-30]}
```